pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -697,6 +697,16 @@ in
     buildFlags = [ "--disable-lto" ];
   });
 
+  sass-embedded = _attrs: {
```

Comment:
@AndersonTorres Re: https://github.com/NixOS/nixpkgs/pull/233655#discussion_r1207541182:
Hm… but throughout the rest of the same file (and throughout the rest of nixpkgs, AFAIK) this convention is not held. I would suggest to align with the prevailing custom and revert it back to `attrs` (no underscore).